### PR TITLE
Export SIRIUS mgf harmonized

### PR DESCRIPTION
**Pull request by robinschmid**
@SteffenHeu I took your branch and added a few changes. I tested it with LC-MS data.

Replaces #351

### CODE REVIEW

File: src/main/java/io/github/mzmine/datamodel/features/ModularFeatureListRow.java [hunk 1]
```diff
@@ -693,6 +693,22 @@ public IsotopePattern getBestIsotopePattern() {
     return null;
   }
 
+
+  @Override
+  public boolean hasIsotopePattern() {
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/datamodel/features/ModularFeatureListRow.java:698]: I theory a row might now have an (invalid) isotope pattern but this function could return false.
I think the addIsotopePattern method should check if the added isotope pattern has more than one data point and not add it, if it doesnt.

**Review by SteffenHeu — COMMENTED** [review]
Is it tested?

File: src/main/java/io/github/mzmine/datamodel/features/ModularFeatureListRow.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/datamodel/features/ModularFeatureListRow.java:698]: good idea

Comment (robinschmid) [reply src/main/java/io/github/mzmine/datamodel/features/ModularFeatureListRow.java:698]: Removed the check here. It is checked in the isotope filter (grouper) and in the finder

**Comment by robinschmid** [comment]
Tested and ready. It is more comparable to the GNPS export now. But we still need this export as SIRIUS needs other input (MS1 etc).

**Pull request merged**